src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,129 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 18;
+    private static final int INVALID_AGE = 14;
+    private static final int NEGATIVE_AGE = -18;
+    private static final int MAX_VALID_AGE = 110;
+    private static final String VALID_LOGIN = "geronimo";
+    private static final String VALID_PASSWORD = "ziggurat_687";
+    private static final String INVALID_PASSWORD = "o7 07";
+    private static final String EMPTY_LINE = "";
+    private static RegistrationService service;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(INVALID_AGE);
+        user.setLogin(EMPTY_LINE);
+        user.setPassword(INVALID_PASSWORD);
```

Comment:
let's create a valid user here and set invalid values directly in your test methods. I think not all of your tests will be successfully passed after that.